content/docs/instrumenting/exporters.md

```diff
@@ -98,6 +98,7 @@ separate exporters are needed:
    * [Etcd](https://github.com/coreos/etcd)
    * [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos)
    * [Kubernetes](https://github.com/GoogleCloudPlatform/kubernetes)
+   * [linkerd](https://github.com/BuoyantIO/linkerd)
```

Comment:
Do you feel strongly about the lower-case "l"? Otherwise, I'd prefer a capital "L"  for consistency.